docs/src/Combinatorics/matroids.md

```diff
@@ -119,7 +119,36 @@ augmented_chow_ring(M::Matroid)
 
 ## Matroid strata and realization spaces
 
-For a matroid ``M``, of rank ``d`` on a ground set ``E`` of size ``n`` realizable over a ring ``K``, 
+
+Let ``M`` be a matroid of rank ``d`` on a ground set ``E`` of size ``n``. Its *realization space* 
+``\mathcal{R}(M)`` is an affine scheme that parameterizes all hyperplane arrangements that realize
+the matroid ``M`` (up to the action of ``PGL(r)``). 
+
+
+```@docs
+is_representable(M; char::Union{Int,Nothing}=nothing, q::Union{Int,Nothing}=nothing)
```

Comment:
I cannot find the implementation of this function.

Reply:
Fixed as indicated above.

Reply:
(should I resolve this?)